inc/namespace.php

```diff
@@ -54,3 +56,30 @@ function load_publication_checklist() {
 
 	require_once Altis\ROOT_DIR . '/vendor/humanmade/publication-checklist/plugin.php';
 }
+
+/**
+ * Load the QM Collector for debugging notifications.
+ */
+function load_query_monitor_debug() {
+
+	if ( ! class_exists( QM_Collectors::class ) ) {
+		return;
+	}
+
```

Comment:
I think we need to check the config value for whether notifications is enabled or not too. If you make the change to use the classmap autoloader you could put the `add_filter()` calls into the `load_workflows()` function to make use of the existing check there.

Reply:
I tried that at first, but I think that might be too early because I didn't get any output?